packages/odyssey/src/scss/components/_tab.scss

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018-Present Okta, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.ods-tabs {
+  --ods-indicator-width: 0;
+  --ods-indicator-pos-x: 0;
+
+  .ods-tablist {
+    position: relative;
+
+    &::before {
+      content: '';
+      position: absolute;
+      z-index: 1;
+      bottom: -1px;
+      left: var(--ods-indicator-pos-x);
+      width: var(--ods-indicator-width);
+      height: 3px;
+      transition: all 0.23s ease-in-out;
+      background: cv('action', 'base');
+    }
+
+    .ods-tab {
+      display: inline-block;
+      position: relative;
+      margin: 0;
+      padding: $small-spacing $base-spacing;
+      border: 0;
+      background: none;
+      font-family: inherit;
+      font-size: ms(0);
+      font-weight: 600;
+      line-height: 1.5;
```

Comment:
You can use `$base-line-height` here.

Reply:
This is actually a remnant of my prototype. It has been removed.